home/models.py

```diff
@@ -1,3 +1,32 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Gender(models.TextChoices):
+    MALE = 'M', 'Male'
+    FEMALE = 'F', 'Female'
+    UNSPECIFIED = 'U', 'Unspecified'
+
+
+class Profile(models.Model):
+    '''
+    Profile will be used as the base user in the Shlifim website.
+    Profile is an extention to the imported 'User' from 'django.contrib.auth.models'.
+    Imported Fields:
+    username - Required. 150 characters or fewer. Usernames may contain alphanumeric, _, @, +, . and - characters.
+    password - Required. A hash of, and metadata about, the password. (Django doesn’t store the raw password).
+               Raw passwords can be arbitrarily long and can contain any character.
+    date_joined - A datetime designating when the account was created. Is set to the current date/time by default
+        when the account is created.
+    email - Email address.
+    is_superuser - Boolean. Designates that this user has all permissions without explicitly assigning them.
+    Added Fields:
+    gender - Default gender is set to 'Unspecified', A user can choose to specifie its gender to Male/Female.
+    is blocked - Boolean. Designates that this user won't be able to login until an admin unblockes him.
+    '''
+    user = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
```

Comment:
While this will work, the best practice is to refer to the AUTH_USER_MODEL.
See: https://learndjango.com/tutorials/django-best-practices-referencing-user-model